### ISSUE TITLE
Cannot escape curly braces in attribute values

### ISSUE THREAD

**Issue by hkan** [maizzle/framework#160]
- Maizzle Version: 1.0.0
- Node.js Version: 14.1.0

Escaping curly braces with `@` in content works as expected. But escaping it between `<` and `>` does not work at all. If I try it inside an attribute value (like `<a href="@{{ config('app.url') }}">`) it keeps `@` as literal symbol and tries to evaluate the rest. This fails and is unexpected for me, and I could not figure out any other way to print `{{ anything }}` inside of an HTML element.

**Comment by cossssmin** [comment]
Yes, just noticed it today as well, it's a bug in `posthtml-expressions`. 

Until that is fixed, use the [`<raw>`](https://maizzle.com/docs/tags/#raw) tag instead:

```html
<raw>
  <a href="{{ config('app.url') }}">Home</a>
</raw>
```

**Comment by cossssmin** [comment]
For others until it gets fixed:

1. Escaping fails inside attributes

This:

```html
<a href="{{ config('app.url') }}">Home</a>
```

Becomes this (or simply fails to evaluate):

```html
<a href="@{{ undefined }}">Home</a>
```

2. Escaping in content is a bit glitchy:

```html
<p>This is some @{{ variable }}, and here is another: @{{ bar }}</p>
```

Becomes:

```html

**Comment by hkan** [comment]
That was a quick answer! Thanks.

I've used `<raw>` to bypass the problem, but the slight issue with `<raw>` is that it literally spews out everything in it as is, as expected. I've converted classes to inline styles myself for that part, but it's not nice to look at now. 😀

**Comment by cossssmin** [comment]
You can keep using Tailwind classes inside the raw tag and Maizzle will inline them - the inliner runs after the templating engine. Have you tried and it didn’t work?

**Comment by hkan** [comment]
My bad, it works! I might have checked the local build and thought it did not do its job.

**Comment by cossssmin** [comment]
Fixed in [v1.0.9](https://github.com/maizzle/framework/releases/tag/v1.0.9)
